db/seeds.rb

```diff
@@ -0,0 +1,58 @@
+post3.comments.create(username: "Arda",    text: "Morbi vel lectus in quam fringilla rhoncus. Mauris enim leo, rhoncus sed, vestibulum sit amet, cursus id, turpis. Integer aliquet, massa id lobortis convallis, tortor risus dapibus augue, vel accumsan tellus nisi eu orci.")
+post4.comments.create(username: "Birch",   text: "Phasellus sit amet erat. Nulla tempus. Vivamus in felis eu sapien cursus vestibulum.")
+post4.comments.create(username: "Dani",    text: "In est risus, auctor sed, tristique in, tempus sit amet, sem. Fusce consequat.")
+post5.comments.create(username: "Marjory", text: "Nunc purus. Phasellus in felis. Donec semper sapien a libero.")
+post5.comments.create(username: "Arnaldo", text: "Nam ultrices, libero non mattis pulvinar, nulla pede ullamcorper augue, a suscipit nulla elit ac nulla. Sed vel enim sit amet nunc viverra dapibus.")
+post6.comments.create(username: "Arda",    text: "Morbi vel lectus in quam fringilla rhoncus. Mauris enim leo, rhoncus sed, vestibulum sit amet, cursus id, turpis. Integer aliquet, massa id lobortis convallis, tortor risus dapibus augue, vel accumsan tellus nisi eu orci.")
+post6.comments.create(username: "Birch",   text: "Phasellus sit amet erat. Nulla tempus. Vivamus in felis eu sapien cursus vestibulum.")
+post7.comments.create(username: "Dani",    text: "In est risus, auctor sed, tristique in, tempus sit amet, sem. Fusce consequat.")
+post7.comments.create(username: "Marjory", text: "Nunc purus. Phasellus in felis. Donec semper sapien a libero.")
+post8.comments.create(username: "Arnaldo", text: "Nam ultrices, libero non mattis pulvinar, nulla pede ullamcorper augue, a suscipit nulla elit ac nulla. Sed vel enim sit amet nunc viverra dapibus.")
+post8.comments.create(username: "Birch",   text: "Phasellus sit amet erat. Nulla tempus. Vivamus in felis eu sapien cursus vestibulum.")
+post9.comments.create(username: "Dani",    text: "In est risus, auctor sed, tristique in, tempus sit amet, sem. Fusce consequat.")
+post9.comments.create(username: "Marjory", text: "Nunc purus. Phasellus in felis. Donec semper sapien a libero.")
+post10.comments.create(username: "Arnaldo", text: "Nam ultrices, libero non mattis pulvinar, nulla pede ullamcorper augue, a suscipit nulla elit ac nulla. Sed vel enim sit amet nunc viverra dapibus.")
+post10.comments.create(username: "Birch",   text: "Phasellus sit amet erat. Nulla tempus. Vivamus in felis eu sapien cursus vestibulum.")
+post10.comments.create(username: "Dani",    text: "In est risus, auctor sed, tristique in, tempus sit amet, sem. Fusce consequat.")
+post10.comments.create(username: "Marjory", text: "Nunc purus. Phasellus in felis. Donec semper sapien a libero.")
+post10.comments.create(username: "Arnaldo", text: "Nam ultrices, libero non mattis pulvinar, nulla pede ullamcorper augue, a suscipit nulla elit ac nulla. Sed vel enim sit amet nunc viverra dapibus.")
```

Comment:
Encourage you to look into the `faker` gem so that creating seed data can be reduced to a couple of `.each` statements.

https://github.com/stympy/faker